frontend/src/app/components/containers/complaints/outcomes/oucome-by-animal/drug-used.tsx

```diff
@@ -218,7 +218,7 @@ export const DrugUsed = forwardRef<refProps, props>((props, ref) => {
     <Card className="comp-drug-form">
       <Card.Header className="comp-card-header px-0">
         <div className="comp-card-header-title">
-          <h5>{drug ? <>{drug}</> : <>Add Drug</>}</h5>
+          <h5>{<>Add Drug</>}</h5>
```

Comment:
Minor change here. The way you have the fix is almost correct, though instead of doing this: ```<h5>{<>Add Drug</>}</h5>```
you're better off doing ```<h5>Add Drug</h5>``` this way react doesn't need to evaluate the contents of the curly braces.

Reply:
Fixed it up now. :-) @marqueone-ps